apps/src/StudioApp.js

```diff
@@ -204,6 +204,10 @@ StudioApp.prototype.configure = function (options) {
   this.assetUrl = _.bind(this.assetUrl_, this);
 };
 
+StudioApp.prototype.hasInstructionsToShow = function (config) {
+  return Boolean(config.level.instructions || config.level.aniGifURL);
+};
```

Comment:
:+1: to taking the config as an argument instead to getting it through some global variable!